### ISSUE TITLE
cannot reproduce the result with CARE-S

### ISSUE THREAD

**Issue by kiyoungkim1** [zengqunzhao/EfficientFace#1]
Thanks for sharing the models.

I try to reproduce the results in the paper with CARE-S dataset.
At first, ```view(-1)``` does not work in the ```accuracy  func.``` (325 lines in main.py),
so I use ```reshape(-1)``` instaed of ```view(-1)```.
I might think it does not affect the results.

Nevertheless,
I can`t get the expected result.
The accuracy is quite low even though I use the pre-trained model uploaded in this repo.
The accuracy of test set is not bad at first time, but it becomes worse with epochs.
The progress is shown below.

``` 
Training time: 05-31 16:52
Current learning rate:  0.1
Epoch: [0][  0/157]     Loss 0.2979 (0.2979)    Accuracy 12.500 (12.500)
Epoch: [0][ 10/157]     Loss 0.2384 (0.2485)    Accuracy 19.531 (13.707)
Epoch: [0][ 20/157]     Loss 0.2250 (0.2420)    Accuracy 10.156 (14.174)
Epoch: [0][ 30/157]     Loss 0.2187 (0.2359)    Accuracy 17.188 (14.793)
Epoch: [0][ 40/157]     Loss 0.2107 (0.2317)    Accuracy 17.969 (15.377)
Epoch: [0][ 50/157]     Loss 0.2052 (0.2272)    Accuracy 23.438 (15.395)
Epoch: [0][ 60/157]     Loss 0.2063 (0.2236)    Accuracy 16.406 (15.151)
Epoch: [0][ 70/157]     Loss 0.2074 (0.2214)    Accuracy 16.406 (15.119)
Epoch: [0][ 80/157]     Loss 0.2119 (0.2199)    Accuracy 11.719 (15.239)
Epoch: [0][ 90/157]     Loss 0.2050 (0.2181)    Accuracy 15.625 (15.393)
Epoch: [0][100/157]     Loss 0.2081 (0.2168)    Accuracy 14.844 (15.316)
Epoch: [0][110/157]     Loss 0.2133 (0.2158)    Accuracy 15.625 (15.280)
Epoch: [0][120/157]     Loss 0.2068 (0.2151)    Accuracy 13.281 (15.283)
Epoch: [0][130/157]     Loss 0.2117 (0.2149)    Accuracy 15.625 (15.357)
Epoch: [0][140/157]     Loss 0.2069 (0.2142)    Accuracy 17.188 (15.376)
Epoch: [0][150/157]     Loss 0.2092 (0.2137)    Accuracy 12.500 (15.444)
Test: [0/7]     Loss 1.0344 (1.0344)    Accuracy 57.031 (57.031)
 *** Accuracy 15.067  *** 
```

**Comment by zengqunzhao** [comment]
> Thanks for sharing the models.
> 
> I try to reproduce the results in the paper with CARE-S dataset.
> At first, `view(-1)` does not work in the `accuracy func.` (325 lines in main.py),
> so I use `reshape(-1)` instaed of `view(-1)`.
> I might think it does not affect the results.
> 
> Nevertheless,
> I can`t get the expected result.
> The accuracy is quite low even though I use the pre-trained model uploaded in this repo.
> The accuracy of test set is not bad at first time, but it becomes worse with epochs.
> The progress is shown below.
> 
> ```
> Training time: 05-31 16:52
> Current learning rate:  0.1
> Epoch: [0][  0/157]     Loss 0.2979 (0.2979)    Accuracy 12.500 (12.500)
> Epoch: [0][ 10/157]     Loss 0.2384 (0.2485)    Accuracy 19.531 (13.707)
> Epoch: [0][ 20/157]     Loss 0.2250 (0.2420)    Accuracy 10.156 (14.174)
> Epoch: [0][ 30/157]     Loss 0.2187 (0.2359)    Accuracy 17.188 (14.793)
> Epoch: [0][ 40/157]     Loss 0.2107 (0.2317)    Accuracy 17.969 (15.377)
> Epoch: [0][ 50/157]     Loss 0.2052 (0.2272)    Accuracy 23.438 (15.395)
> Epoch: [0][ 60/157]     Loss 0.2063 (0.2236)    Accuracy 16.406 (15.151)
> Epoch: [0][ 70/157]     Loss 0.2074 (0.2214)    Accuracy 16.406 (15.119)
> Epoch: [0][ 80/157]     Loss 0.2119 (0.2199)    Accuracy 11.719 (15.239)
> Epoch: [0][ 90/157]     Loss 0.2050 (0.2181)    Accuracy 15.625 (15.393)
> Epoch: [0][100/157]     Loss 0.2081 (0.2168)    Accuracy 14.844 (15.316)
> Epoch: [0][110/157]     Loss 0.2133 (0.2158)    Accuracy 15.625 (15.280)
> Epoch: [0][120/157]     Loss 0.2068 (0.2151)    Accuracy 13.281 (15.283)
> Epoch: [0][130/157]     Loss 0.2117 (0.2149)    Accuracy 15.625 (15.357)
> Epoch: [0][140/157]     Loss 0.2069 (0.2142)    Accuracy 17.188 (15.376)
> Epoch: [0][150/157]     Loss 0.2092 (0.2137)    Accuracy 12.500 (15.444)
> Test: [0/7]     Loss 1.0344 (1.0344)    A

**Comment by zengqunzhao** [comment]
Thank you for your interest in our work. 

Do you mean CAER-S dataset? 
you should change the learning rate, factor, and Adjust-freq.

**Comment by zengqunzhao** [comment]
Towards different datasets, those hype-parameters may a little different. I will show you later. :)

**Comment by kiyoungkim1** [comment]
Thanks!
quite interesting that the model gets a high score with only a few params.

> Towards different datasets, those hype-parameters may a little different. I will show you later. :)
Does this mean that model is quite sensitive to the dataset?
simple adjustment of learning rate may not be a solution for my problem.
I also adjust the ```transform.normalize```, but it also does not a proper remedy.

I have to try other things.

**Comment by zengqunzhao** [comment]
I just forgot one important thing, that is you should first fine-tune the LDG(Resnet50) on corresponding dataset. The Pre-trained LDG model which I provide has been fine-tuned on RAF-DB. 
You can try it.

**Comment by TheKOne** [comment]
您好，请问在CAER-S数据集上的超参数设置大概是怎样呢？谢谢

**Comment by zengqunzhao** [comment]
> 您好，请问在CAER-S数据集上的超参数设置大概是怎样呢？谢谢

Regarding the CAER-S dataset, you can try with:
---epochs: 50
---batch-size: 128
---learning-rate: 0.001
---factor: 0.1
---adjust-freq: 30
